enclave/ecalls.cpp

```diff
@@ -36,9 +36,9 @@ int enclave_XGDMatrixCreateFromFile(const char *fname, int silent, DMatrixHandle
   return XGDMatrixCreateFromFile(fname, silent, out);
 }
 
-int enclave_XGDMatrixCreateFromEncryptedFile(const char *fname, int silent, DMatrixHandle *out) {
+int enclave_XGDMatrixCreateFromEncryptedFile(const char *fname, int silent, DMatrixHandle *out, char* username) {
   LOG(DEBUG) << "Ecall: XGDMatrixCreateFromEncryptedFile";
-  return XGDMatrixCreateFromEncryptedFile(fname, silent, out);
+  return XGDMatrixCreateFromEncryptedFile(fname, silent, out,username);
```

Comment:
Please add a space here between the comma and `username`

Reply:
done